core/src/test/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ContainerOrchestratorResolverTest.java

```diff
@@ -0,0 +1,63 @@
+package com.sequenceiq.cloudbreak.core.bootstrap.service;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import com.sequenceiq.cloudbreak.core.CloudbreakException;
+import com.sequenceiq.cloudbreak.orchestrator.ContainerOrchestrator;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ContainerOrchestratorResolverTest {
+
```

Comment:
This doesn't strictly belongs to the test: in the ContainerOrchestratorResolver class, get method, remove the second containerOrchestrators.get call, and return the co (the firstly got object) instead